bookkeeper-server/src/main/java/org/apache/bookkeeper/client/BookKeeper.java

```diff
@@ -1415,4 +1378,48 @@ static EventLoopGroup getDefaultEventLoopGroup() {
             return new NioEventLoopGroup(numThreads, threadFactory);
         }
     }
+
+    @Override
+    public CreateBuilder createLedgerOp() {
```

Comment:
nit: better naming for this method..

newCreateLedgerOpBuilder
newCreateLedgerBuilder
newCreateBuilder

just like OrderedSafeExecutor.newBuilder()

Reply:
I changed them to newCreateLedgerOp()...the idea is that these are "operations"

/cc @sijie @ivankelly

Reply:
`newCreateLedgerOp` works for me.